jujugui/static/gui/src/app/app.js

```diff
@@ -1954,6 +1955,18 @@ YUI.add('juju-gui', function(Y) {
     },
 
     /**
+      This method is called after the env has been logged into and calls a
```

Comment:
I guess this is small and local enough to be defined firectly as a closure inside switchEnv. [shrug]